packages/gatsby-plugin-inferno/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "gatsby-plugin-inferno",
+  "description": "A Gatsby plugin which replaces React with Inferno",
+  "version": "1.0.0",
+  "author": "Kyle Mathews <mathews.kyle@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/gatsbyjs/gatsby/issues"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.0.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.0.0",
+    "@babel/core": "^7.0.0",
+    "cross-env": "^5.1.4"
+  },
+  "homepage": "https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-plugin-inferno#readme",
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "inferno"
+  ],
+  "license": "MIT",
+  "main": "index.js",
+  "peerDependencies": {
+    "gatsby": ">2.0.0-alpha"
```

Comment:
We would need `inferno` and  `inferno-compat` as peerDeps too

Reply:
Would something like this work?

```
  "peerDependencies": {
    "inferno-compat": "6.x",
    "gatsby": ">2.0.0-alpha"
},
```

It's only Inferno 6 that works at this point